release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -3216,4 +3216,28 @@ $ oc adm release info 4.9.48 --pullspecs
 [id="ocp-4-9-48-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-49"]
+=== RHBA-2022:6678 - {product-title} 4.9.49 bug fix update
+
+Issued: 2022-09-29
+
+{product-title} release 4.9.49 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:6678[RHBA-2022:6678] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:6677[RHBA-2022:6677] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.9.49 --pullspecs
+----
+
+[id="ocp-4-9-49-bug-fixes"]
+==== Bug fixes
+
+* Previously, routers in the terminating state delayed the `oc cp` command, which caused a delay in the `must-gather`. With this update, a timeout for each `oc cp` command has been set, eliminating the delay in the `must-gather`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2108892[*BZ#2108892*])
```

Comment:
Should it be `must-gather` logs? So:
```suggestion
* Previously, routers in the terminating state delayed the `oc cp` command, which caused a delay in the `must-gather` logs. With this update, a timeout for each `oc cp` command has been set, eliminating the delay in the `must-gather` logs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2108892[*BZ#2108892*])
```

Reply:
Hey Olivia! Thanks for the review and you are right! I found logos and sequence after must gather, but more so logos and feel that is right! I have made the changes. :)